test/functional/apps/dashboard_elements/controls/common/range_slider.ts

```diff
@@ -134,6 +134,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         expect(await saveButton.isEnabled()).to.be(false);
         await dashboardControls.controlsEditorSetfield('dayOfWeek');
         await dashboardControls.controlsEditorSetControlType(RANGE_SLIDER_CONTROL);
+        await dashboardControls.rangeSliderEditorSetStep(5);
```

Comment:
**nit** Do we actually want to test this to ensure the `step` is working as intended? Not sure if adding this here is actually... testing anything 🤔